customer-churn/steps/src/utils.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+
+def unique_data_detector(data: pd.DataFrame) -> pd.Series:
+    """Detects unique values in a dataframe.
+
+    Args:
+        data (pd.DataFrame): Dataframe to be analyzed.
```

Comment:
Try to be consistent in how you use words. Probably easiest if you just use DataFrame everywhere you use it.

Reply:
Yeah, Sure